components/eam/src/dynamics/se/inidat.F90

```diff
@@ -390,7 +409,7 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
           ! Since the rest of processing uses tmp, copy qtmp into tmp
           do ie = 1, nelemd
             do k=1,nlev
-              do i = 1, npsq
+             do i = 1, npsq
```

Comment:
Sorry this is nit picky but you should keep the same indent level here.

Reply:
agreed; fixed